### ISSUE TITLE
No stats about fees

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#346]
On the stats page we show how much money entered and left Liberapay each week, but we don't show how much was paid in fees.

**Comment by Changaco** [comment]
Stats about fees could also be added to the FAQ. For example we could display the average fee percentage for all payments processed in the last year. A quick attempt to compute that number gives me 2.79% for payments through Stripe:

```sql
select sum(fee, 'EUR') as fees, sum(amount_settled, 'EUR') as amounts, (sum(fee, 'EUR')).amount / (sum(amount_settled, 'EUR')).amount * 100 as average_fee_percentage from payins where fee is not null;
     fees      |    amounts     | average_fee_percentage 
---------------+----------------+------------------------
 (1214.27,EUR) | (43561.29,EUR) | 2.78749779907803465000
(1 row)
```

**Comment by trebmuh** [comment]
That sounds like a great idea to help user to figure out how much the fees will (roughly) be.

**Comment by Changaco** [comment]
A quick attempt to compute the same number for PayPal: 4.90%.

```sql
select sum(fee, 'EUR') as fees, sum(amount, 'EUR') as amounts, (sum(fee, 'EUR')).amount / (sum(fee, 'EUR') + sum(amount, 'EUR')).amount * 100 as average_fee_percentage from payin_transfers where fee is not null;
     fees      |     amounts     | average_fee_percentage 
---------------+-----------------+------------------------
 (5602.96,EUR) | (108664.79,EUR) | 4.90336074701742180100
(1 row)
```

**Comment by corneliusroemer** [comment]
Yes, that's a good thing to do in aggregate. BUT, it doesn't replace fine-grained fee information. Paypal could take a high fixed fee, so is definitely bad for large amounts, etc.  Sure, you don't want to overwhelm donors with worries about fees (even though I crave this information, it probably actually puts me off donating when I see fees of 5%) - you could over them in a tooltip or accordeon/unfold.

**Comment by trebmuh** [comment]
> On the stats page we show how much money entered and left Liberapay each week, but we don't show how much was paid in fees.

I agree that average fees through paypal and stripe could be a very good informations to have on the [stats page](https://liberapay.com/about/stats).


> it doesn't replace fine-grained fee information

It is my understanding that such calculation are very difficult to do before a transfer is processed by paypal or stripe. As one example: NICEUSER want to make a 5€ donation to the TEAMAWESOME. In TEAMAWESOME, there are several people coming from different countries. From NICEUSER perspective, how to know how much are the fees to be applied by paypal/stripe if we don't know yet on which member of the TEAMAWESOME the donation will be directed (since IIRC, paypal/stripe do not charge the same fees for all countries)?

**Comment by corneliusroemer** [comment]
>  In TEAMAWESOME, there are several people coming from different countries. From NICEUSER perspective, how to know how much are the fees to be applied by paypal/stripe if we don't know yet on which member of the TEAMAWESOME the donation will be directed (since IIRC, paypal/stripe do not charge the same fees for all countries)?

I thought payout is always via stripe?

It's a good point, payout fees are dependent on how much donors withdraw. So maybe an average or range is enough there.

The payout fee should also almost always be smaller than pay in fee, because payouts are presumably larger than most payins.

**Comment by trebmuh** [comment]
> I thought payout is always via stripe?

It is my understanding that:
- if the donator is using paypal, then it's going out with paypal
and
- if the donator is using a bankcard or a SEPA transfer, then it's going out with stripe

**Comment by Changaco** [comment]
Liberapay doesn't do payouts, it merely helps transfer the funds from donors into the Stripe or PayPal accounts of the recipients. Where the money goes after that isn't Liberapay's concern.

**Comment by corneliusroemer** [comment]
Thanks @Changaco, that statement of yours is very clear.
So relevant fees are from donor to recipient account. For that, there could be hard fee information.
